docs/code-review/README.md

```diff
@@ -0,0 +1,25 @@
+# Code Review
```

Comment:
IMO, let's move the Code Review section to the guides instead, as it affects the whole project rather than just integration with Cloud.

Reply:
Per our last discussion, I thought the idea was to work the much more statement / factual guides into modules, and then Guides are more general / solving a problem type stuff.